docs/2.develop/deploy.md

```diff
@@ -23,4 +23,12 @@ By default the contract will be deployed to the testnet. To deploy into `mainnet
 
 :::tip
 You can use `near dev_deploy` to deploy the contract into a newly created account!
-:::
\ No newline at end of file
+:::
+
+<blockquote class="lesson">
```

Comment:
We should transform this into a sub-section talking about what happens when you deploy a contract into an account that already has a contract.